content/applications/inventory_and_mrp/maintenance/maintenance_calendar.rst

```diff
@@ -0,0 +1,378 @@
+
+There are three different types of graphs available to users to view the data:
+
+- :guilabel:`Bar Chart`: the data is displayed in a bar chart.
+- :guilabel:`Line Chart`: the data is displayed in a line chart.
+- :guilabel:`Pie Chart`: the data is displayed in a pie chart.
+
+When viewing the data as a :guilabel:`Bar Chart` graph, the data can be formatted in the following
+ways:
+
+- :guilabel:`Stacked`: the data is stacked on the graph.
+- :guilabel:`Descending`: the data is displayed in descending order.
+- :guilabel:`Ascending`: the data is displayed in ascending order.
+
+When viewing the data as a :guilabel:`Line Chart` graph, the data can be formatted in the following
+ways:
+
+- :guilabel:`Stacked`: the data is stacked on the graph.
+- :guilabel:`Cumulative`: the data is increasingly accumulated.
+- :guilabel:`Descending`: the data is displayed in descending order.
+- :guilabel:`Ascending`: the data is displayed in ascending order.
+
+When viewing the data as a :guilabel:`Pie Chart` graph, all relevant data is displayed by default,
+and no additional formatting options are available.
+
+Activity view
+~~~~~~~~~~~~~
+
+With the :guilabel:`Activity` view selected, all open maintenance requests are listed in their own
+row, with the ability to schedule activities related to those requests.
+
+.. image:: maintenance_calendar/maintenance-calendar-activity-view-type.png
+   :align: center
+   :alt: Maintenance requests on Activity view.
+
+Maintenance requests are listed in the :guilabel:`Maintenance Request` column as activities.
+Clicking a request opens a :guilabel:`Maintenance Request` popover that indicates the status of the
+request and the responsible *Technician*. To schedule an activity directly from the popover, click
+:guilabel:`Schedule an activity` (with the :guilabel:`➕ (plus)` icon). This opens a
+:guilabel:`Schedule Activity` pop-up window.
+
+From the pop-up window, choose the :guilabel:`Activity Type`, provide a :guilabel:`Summary`,
+schedule a :guilabel:`Due Date`, and choose the responsible user in the :guilabel:`Assigned to`
+field.
+
+.. image:: maintenance_calendar/maintenance-calendar-schedule-activity-popover.png
+   :align: center
+   :alt: Schedule Activity pop-up window.
+
+Type any additional notes for the new activity in the blank space under the greyed-out
+:guilabel:`Type "/" for commands`.
+
+Once ready, click :guilabel:`Schedule` to schedule the activity. Alternatively, click
+:guilabel:`Schedule & Mark as Done` to close the activity, click :guilabel:`Done & Schedule Next` to
+close the activity and open a new one, or click :guilabel:`Cancel` to cancel the activity.
+
+With the :guilabel:`Activity` view selected, each *Activity Type* available when scheduling an
+activity is listed as its own column. These columns are :guilabel:`Email`, :guilabel:`Call`,
+:guilabel:`Meeting`, :guilabel:`To-Do`, :guilabel:`Upload Document`, :guilabel:`Request Signature`,
+and :guilabel:`Grant Approval`.
```

Comment:
what about 'Maintenance Request' ... or does that not technically count as an 'Activity Type' -- worth a double-check, for sure

Reply:
It definitely does, added it in.  Suggestion implemented ✅